.yarn/versions/6de667dc.yml

```diff
@@ -0,0 +1,3 @@
+declined:
+  - primitives
+  - "@radix-ui/react-menu-bar"
```

Comment:
teehee, I thought the same as you but when I googled the pattern it seems it's actually one word 🙈

`react-menubar`
`const Menubar = React.forwardRef();`

etc.

Reply:
yep. I noticed that haha. Only had time to change the PR title 😂

Reply:
Excited for this! Thanks again 🙂